op-chain-ops/script/README.md

```diff
@@ -0,0 +1,139 @@
+# Forge Scripts in Go
+
+This package provides Forge scripting functionality in Go. It allows you to write scripts in Solidity, but call them
+from Go using an in-memory EVM. This is useful for building tooling, since it eliminates the need to shell out to Forge
+or install the Solidity toolchain.
+
+# Usage
+
+To use the Forge scripts, you'll need to create a script host. A minimal example of how to do this below:
+
+```go
+artifactsFS := foundry.OpenArtifactsDir("path/to/forge-artifacts")
+host := NewHost(logger, artifactsFS, nil, DefaultContext, nil)
+```
+
+You can pass in optional arguments to `NewHost` to customize its behavior:
+
+- `WithBroadcastHook` will allow you to broadcast the transactions initiated using `vm.broadcast` on chain via an
+  external broadcaster.
+- `WithIsolatedBroadcast` will clear the warm state between calls, which allows broadcasted gas estimates to be more
+  accurate.
+- `WithCreate2Deployer` will configure the `CREATE2` opcode to broadcast transactions to the Arachnid `Create2Deployer`.
```

Comment:
This only affects create2 operations from the script contract itself right, and create2 calls from OPCM are unaffected?

If so, similar to https://github.com/ethereum-optimism/optimism/pull/12056/files?diff=unified&w=0#r1771432789, when would you want this to be false?